__tests__/app.test.js

```diff
@@ -32,4 +35,17 @@ describe('testing /api/topics', () => {
         return request(app).get("/api/randomstuff4")
         .expect(404)
     });
+});
+
+describe('testing GET /api', () => {
+    test('should respond with a 200 status code', () => {
+        return request(app).get("/api")
+        .expect(200)
+    });
+    test('should respond with an object describing all the available endpoints on the API', () => {
+        return request(app).get("/api")
+        .then(response => {
+            expect(allEndpoints).toEqual(response._body);
+        })
```

Comment:
This is fine :) 

As this is an integration test, you're free to actually just combine this into 1 test which asserts that you get `200` status code returned, along with checking the response is equal to the `json` file, no need for 2 separate tests here.